### PR TITLE
User-centric Teams messaging subscription documentation

### PR DESCRIPTION
**Pull request by eddie-lee-msft**
User-centric Teams messaging subscription documentation

### CODE REVIEW

**Review by RamjotSingh — CHANGES_REQUESTED** [review]

File: concepts/teams-changenotifications-chatmessage.md [hunk 1]
```diff
@@ -252,6 +252,58 @@ Content-Type: application/json
 }
 ```
 
+## Subscribe to changes at the user level
+
+To track messages for all chats a user is part of, you can create a change notification subscription for all chats at the user level. To do this, subscribe to `/users/{user-id}/chats/getAllMessages` or `/me/chats/getAllMessages` if creating the subscription in delegated mode. This resource supports [including resource data](webhooks-with-resource-data.md) in the notification in both *delegated* and *application-only modes*.
+
+User-level chat messaging subscriptions also support keyword-based search via the `$search` query parameter.
+
+> **Note:** User-level chat messaging subscriptions are currently in preview.
```
Comment (RamjotSingh) [inline concepts/teams-changenotifications-chatmessage.md:261]: I believe 2 Note back to back are not liked. You can remove first Note since we mention it below.

**Review by RamjotSingh — APPROVED** [review]

File: concepts/teams-changenotifications-chatmessage.md [hunk 2]
```diff
@@ -252,6 +252,56 @@ Content-Type: application/json
...
+  "notificationUrl": "https://webhook.azurewebsites.net/api/resourceNotifications",
+  "resource": "/users/{user-id}/chats/getAllMessages",
+  "includeResourceData": true,
+  "encryptionCertificate": "{base64encodedCertificate}",
+  "encryptionCertificateId": "{customId}",
+  "expirationDateTime": "2019-09-19T11:00:00.0000000Z",
+  "clientState": "{secretClientState}"
+}
+```
+### Example 2: Subscribe to messages in all chats a user is part of in delegated mode, using the `/me/chats/getAllMessages` resource path
```
[the remark this answers is not in the document]
Comment (eddie-lee-msft) [reply concepts/teams-changenotifications-chatmessage.md:288]: Actually, at the user level '/me/chats/getAllMessages' is not required. It is simply another option folks can use in addition to '/users/user-id/chats/getAllMessages' when calling in delegated mode.

[the remark this answers is not in the document]
Comment (eddie-lee-msft) [reply concepts/teams-changenotifications-chatmessage.md:288]: What would you suggest I rephrase this to in this case, as well as in the blob from your comment above?

**Review by JarbasHorst — CHANGES_REQUESTED** [review]

**Comment by JarbasHorst** [comment]
@Lauragra this PR is ready for final review.

**Review by Lauragra — APPROVED** [review]
